packages/pigment-css-vite-plugin/src/vite-plugin.ts

```diff
@@ -132,17 +132,19 @@ export default function wywVitePlugin({
         .filter((m): m is ModuleNode => !!m);
     },
     async transform(code, url) {
-      const [id] = url.split('?', 1);
-
+      const [filePath] = url.split('?', 1);
```

Comment:
How does this arg differs from the one in the unplugin (url vs filePath)?

Reply:
No difference. Updated to use same names.